src/pages/HomePage.api.test.tsx

```diff
@@ -0,0 +1,94 @@
+import { render, screen, waitFor } from '@testing-library/react';
```

Comment:
same for this file, just move missing part to HomePage.test.tsx file and delete this file